Module_Java_ReflectionApi/src/main/java/pages/SearchResultsPage.java

```diff
@@ -0,0 +1,67 @@
+package pages;
+
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+
+public class SearchResultsPage extends BasePage {
+
+    @FindBy(xpath = "//div[contains(@data-cel-widget, 'search_result_')]")
+    private List<WebElement> productList;
+
+    @FindBy(xpath = "//li[@aria-label='HP']//i[@class='a-icon a-icon-checkbox']")
+    private WebElement hpMonitorsPage;
+
+    @FindBy(xpath = "//span[@class='a-size-base-plus a-color-base a-text-normal']")
+    private List<WebElement> productContentList;
+
+    @FindBy(xpath = "//*[@class='a-section aok-relative s-image-square-aspect']")
+    private List<WebElement> imageBlockList;
+
+    public SearchResultsPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public boolean isProductContentListDisplayed() throws InterruptedException {
+        Thread.sleep(2000);
+        return productContentList.get(0).isDisplayed();
+    }
+
+    public void checkThatSearchResultsContainsWord() throws InterruptedException {
+        Thread.sleep(2000);
+        for (WebElement webElement : productContentList) {
+            Assert.assertTrue(webElement.getText().contains("HP"));
```

Comment:
Avoid hardcoding please. Create variable for it

Reply:
> Avoid hardcoding please. Create variable for it

Done